### PR TITLE
feat: Use ose-tools to get kubectl and enable hermetic builds

### PR DESCRIPTION
**Pull request by flacatus**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -41,25 +41,19 @@ LABEL \
 
 WORKDIR /rhtap-cli
 
+COPY --from=ose-tools /usr/bin/kubectl /usr/bin/
+
 COPY --from=builder /workdir/rhtap-cli/installer ./
 
 COPY --from=builder /workdir/rhtap-cli/bin/rhtap-cli /usr/local/bin/rhtap-cli
+COPY --from=builder /usr/bin/tar /usr/bin/
```
Comment (Roming22) [inline Dockerfile]: 'tar' & 'gzip' not needed if we copy 'kubectl' from an image.

**Review by Roming22 — APPROVED** [review]
/lgtm

**Review by Roming22 — APPROVED** [review]

**Comment by Roming22** [comment]
@flacatus this PR is conflicting with #223. Please close the one that is obsolete, and confirm on the one that is left that it is ready to be merged.

**Comment by flacatus** [comment]
@Roming22 this is the right pr

**Pull request merged**